### PR TITLE
fix: Content set, Safari 14.1.1. / WebKitGTK 2.32.0

### PR DESCRIPTION
**Pull request by castastrophe**
## <Pull request description>

Safari Version 14.1.1 (16611.2.7.1.4) (WebKitGTK 2.32.0) is throwing a blocking page error on this line of code: `else this.shadowRoot.querySelector(`#container`).innerHTML = view.outerHTML;`

A bug has been filed against WebKit (https://bugs.webkit.org/show_bug.cgi?id=224408) and it appears to be resolved in the preview version.

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Preview page](https://deploy-preview-1650--patternfly-elements.netlify.app/elements/pfe-content-set/demo) 

### Testing instructions

- [x] Open [this link](https://deploy-preview-1650--patternfly-elements.netlify.app/elements/pfe-content-set/demo) in latest Safari (14.1.1), does it load? :)

#### Regression validation

Your component should work in all of the following environments:

- [x] Latest 2 versions of Edge
- [x] Internet Explorer 11 (should be useable, not pixel perfect)
- [x] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [x] Firefox 78 (or latest version for Red Hat Enterprise Linux distribution)
- [x] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [x] Latest 2 versions of Safari
- [x] Android mobile device (such as the Galaxy S9)
- [x] Apple mobile device (such as the iPhone X)
- [x] Apple tablet device (such as the iPhone Pro)

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Browser testing passed.
- [x] Changelog updated.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

**Review by zhawkins — APPROVED** [review]
Looks good to me!

File: elements/pfe-content-set/src/pfe-content-set.js [hunk 1]
```diff
@@ -175,7 +175,7 @@ class PfeContentSet extends PFElement {
   get view() {
     if (!this._rendered) return;
 
-    return this.shadowRoot.querySelector(`#container > ${this.expectedTag}`);
+    return this.shadowRoot.querySelector(`#container ${this.expectedTag}`);
```
Comment (castastrophe) [inline elements/pfe-content-set/src/pfe-content-set.js:178]: @zhawkins Here be my container issue

File: elements/pfe-content-set/src/pfe-content-set.js [hunk 2]
```diff
@@ -252,7 +252,7 @@ class PfeContentSet extends PFElement {
     this._updateBreakpoint = this._updateBreakpoint.bind(this);
 
     this._observer = new MutationObserver(this._mutationHandler);
-    if (window.ResizeObserver) this._resizeObserver = new ResizeObserver(this._resizeHandler);
+    // if (window.ResizeObserver) this._resizeObserver = new ResizeObserver(this._resizeHandler);
```
Comment (castastrophe) [inline elements/pfe-content-set/src/pfe-content-set.js:255]: This bad-boy was failing

File: elements/pfe-content-set/src/pfe-content-set.js (hunk 1 above)
Comment (zhawkins) [reply elements/pfe-content-set/src/pfe-content-set.js:178]: Ah, I see.

File: elements/pfe-content-set/src/pfe-content-set.js (hunk 2 above)
Comment (zhawkins) [reply elements/pfe-content-set/src/pfe-content-set.js:255]: Can you add comments around the commented out code sections referencing the Webkit bug?

**Review by zhawkins — APPROVED** [review]
Needs comments and maybe a debounce on the window resize.

**Pull request merged**